app/serializers/form_template_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class FormTemplateSerializer < ApplicationSerializer
```

Comment:
Do you anticipate we will use this serializer outside of admin routes? If not, then I'd recommend prepending `Admin` to the serializer name and file here.